include/psp2/sysmodule.h

```diff
@@ -90,9 +90,17 @@ enum {
 	SCE_SYSMODULE_AVPLAYER              = 0x004c
 };
 
+/* internal module IDs */
+enum {
```

Comment:
Could you change it to a `typedef struct` and give it a name? This is easier for making a reference to it in the documentation.

Also multi-line doxygen-style comments start with `/**` (notice the 2 stars).